recipes/boost/all/conandata.yml

```diff
@@ -278,3 +278,5 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/1.79.0-smart_ptr_cw_ppc_msync.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/1.79.0-geometry_no_rtti.patch"
+      base_path: "source_subfolder"
```

Comment:
Please, add some more meta-information to this patch following the documentation that @danimtb is adding. We are adding more docs to [this section about patching](https://github.com/conan-io/conan-center-index/blob/master/docs/policy_patching.md) in this PR (https://github.com/conan-io/conan-center-index/pull/11713).

 Here it would be handy to add at least:
  * `patch_type`: probably this one is about "portability"
  * `url`: if there is a link to this patch
  * `patch_source`: if there is a link to some issue or mailing list where this patch was discussed.
  
  
 Thanks!

Reply:
Nice! Thanks for pointing that out to me. That metadata will be really helpful. I've updated my PR to include that information.

Reply:
@jgsogo Looks like that's the straw that broke the camel's back:

`ERROR: [RECIPE FOLDER SIZE (KB-H009)] The size of your recipe folder (256.0615234375 KB) is larger than the maximum allowed size (256KB). (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H009)`

What should I do?

Reply:
Nvm, I removed the version 1.69.0 of Boost to reduce the size of the recipe.